src/helpers/conditions/user.js

```diff
@@ -9,6 +9,8 @@ export const hasRole = (role) => ({ currentUser }) => currentUser.access_level =
 
 export const isAdmin = any(hasRole('admin'), hasRole('superuser'));
 
+export const isSubaccountUser = ({ currentUser }) => currentUser.access_level && currentUser.access_level.includes('subaccount');
```

Comment:
Can do a nested destructure here. Also note that if `access_level` does not exist, it will return `undefined`. It's still falsy, but the other functions return the boolean `false`. Shouldn't be an issue though. 
```suggestion
export const isSubaccountUser = ({ currentUser : {access_level} }) => access_level && access_level.includes('subaccount');
```